pkg/kubelet/prober/results/results_manager_test.go

```diff
@@ -97,3 +97,65 @@ func TestUpdates(t *testing.T) {
 	m.Set(barID, Success, pod)
 	expectUpdate(Update{barID, Success, pod.UID}, "changed bar")
 }
+
+func TestResult_ToPrometheusType(t *testing.T) {
+	tests := []struct {
+		name string
+		r    Result
+		want float64
```

Comment:
nit: try to follow naming conventions used in this module, e.g. want -> expected, tt -> test etc.